packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -523,6 +523,9 @@ class EditableTextState extends State<EditableText> implements TextInputClient {
   @override
   Widget build(BuildContext context) {
     FocusScope.of(context).reparentIfNeeded(widget.focusNode);
+    Scrollable.of(context)?.widget?.controller?.addListener((){
+      _selectionOverlay?.hide();
+    });
```

Comment:
@Hixie: Do you think that something like this could work as an workaround?
https://github.com/otaviojr/flutter/commit/2c49088e328d71904dbaeb11f6757c6c33ea9af4